endpoints/cookie_sync.go

```diff
@@ -20,20 +20,25 @@ import (
 	"github.com/prebid/prebid-server/usersync"
 )
 
-func NewCookieSyncEndpoint(syncers map[openrtb_ext.BidderName]usersync.Usersyncer, hostCookie *config.HostCookie, syncPermissions gdpr.Permissions, metrics pbsmetrics.MetricsEngine, pbsAnalytics analytics.PBSAnalyticsModule) httprouter.Handle {
+func NewCookieSyncEndpoint(syncers map[openrtb_ext.BidderName]usersync.Usersyncer, cfg *config.Configuration, syncPermissions gdpr.Permissions, metrics pbsmetrics.MetricsEngine, pbsAnalytics analytics.PBSAnalyticsModule) httprouter.Handle {
 	deps := &cookieSyncDeps{
 		syncers:         syncers,
-		hostCookie:      hostCookie,
+		hostCookie:      &cfg.HostCookie,
+		gDPR:            &cfg.GDPR,
 		syncPermissions: syncPermissions,
 		metrics:         metrics,
 		pbsAnalytics:    pbsAnalytics,
 	}
 	return deps.Endpoint
 }
 
+var one = 1
```

Comment:
I know what you wrote is more efficient, but... I think you should just move these inside the `Endpoint()`.

It would be very easy for someone to write `parsedReq.GDPR = {thing}` and accidentally change the value of this global "constant". A bug like that would be nearly impossible to track down.